metron-platform/metron-writer/src/test/java/org/apache/metron/writer/BulkWriterComponentTest.java

```diff
@@ -118,6 +118,20 @@ public void writeShouldProperlyAckTuplesInBatch() throws Exception {
   }
 
   @Test
+  public void writeShouldProperlyAckTuplesWhenWriterDisabled() throws Exception {
```

Comment:
It's not a huge deal, but is it possible to write a test where one writer is enabled and one is disabled?  I'm mostly concerned about making sure everything is spelled out, to avoid any regressions if anything in here gets refactored.